infrastructure/helm-chart/charts/api-auth/templates/deployment.yaml

```diff
@@ -22,29 +22,28 @@ spec:
     spec:
       containers:
       - name: app
-        image: ghcr.io/airyhq/api/auth:latest
+        image: "{{ .Values.image }}:{{ .Values.global.appImageTag }}"
         imagePullPolicy: Always
+        envFrom:
+          - configMapRef:
+              name: user-config
         env:
         - name: SERVICE_NAME
           value: api-auth
         - name: DB_USERNAME
           value: postgresadmin
         - name: DB_PASSWORD
-          value: <pg_password>
+          valueFrom:
+            configMapKeyRef:
+              name: postgres-config
+              key: POSTGRES_PASSWORD
         - name: DB_ENDPOINT
           value: postgres:5432
         - name: JWT_SECRET
-          value: <jwt_secret>
-        - name: MAIL_URL
-          value: <mail_url>
-        - name: MAIL_PORT
-          value: "<mail_port>"
-        - name: MAIL_USERNAME
-          value: <mail_username>
-        - name: MAIL_PASSWORD
-          value: <mail_password>
-        - name: MAIL_FROM
```

Comment:
those are requirements for the api auth application so, unless I'm not getting how this works now, the pod will never reach running state with this deployment descriptor?

Reply:
If you look up a couple of lines the whole user-config cm is used as an env input. Which works but I get that explicit is better than implicit since maybe in the future there will be vars in the user-config that should not be in this pod